src/Pos/Binary/Txp.hs

```diff
@@ -21,6 +21,10 @@ instance Bi T.TxOut where
     put (T.TxOut addr coin) = put addr >> put coin
     get = label "TxOut" $ T.TxOut <$> get <*> get
 
+instance Bi T.TxOutAux where
+    put (T.TxOutAux out distr) = put out >> put distr
```

Comment:
Ok, here is a minor issue https://issues.serokell.io/issue/CSL-896
I don't think it makes sense to do something with it in next few months, but maybe after that…